Prox/Prox/Widgets/ReviewContainerView.swift

```diff
@@ -106,6 +115,8 @@ class ReviewContainerView: UIView {
 
         constraints += [bottomAnchor.constraint(equalTo: numberOfReviewersLabel.bottomAnchor)]
 
+        collapsedConstraints = [ heightAnchor.constraint(equalToConstant: 0) ]
```

Comment:
I'll add a comment mentioning this as a shortcoming of the `isCollapsed` setter if you're just concerned about future code.